dev_guide/cron_jobs.adoc

```diff
@@ -75,9 +75,77 @@ spec:
           - name: pi
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: Never
+          restartPolicy: Never <3>
 ----
 
-1. Schedule for the job. In this example, a job will run every minute.
+1. Schedule for the job. In this example, the job will run every minute.
 2. Job template. This is similar to the xref:../dev_guide/jobs.adoc#creating-a-job[job example].
+3. The restart policy of the pod. This does not apply to the job controller. See xref:../dev_guide/jobs.adoc#creating-a-job-known-issues[Known Issues and Limitations] for details.
 ====
+
+Alternatively, you can create and launch a cron job from a single command -- specifically, `oc run`. The following command creates and launches the same cron job as the `pi` resource:
+
+----
+$ oc run pi --image=perl --schedule='* * * * *' --restart=Never --command -- perl -Mbignum=bpi -wle 'print bpi(2000)' 
+----
+ 
+With `oc run`, the `--schedule` option accepts schedules in link:https://en.wikipedia.org/wiki/Cron[cron format]. 
+
+[NOTE]
+=====
+When creating a cron job,  `oc run` only supports the `Never` or `OnFailure` restart policies (`--restart`).
+=====
+
+[TIP]
+=====
+Delete cron jobs that you no longer need:
+----
+$ oc delete --cronjob <cron_job_name>
+----
+Doing so will prevent them from generating unnecessary artifacts.
+=====
+
+[[cleaning-up-after-a-cron-job]]
+== Cleaning Up After a Cron Job
+
+Cron jobs can leave behind artifact resources such as jobs, pods, or deployment configurations. Check if any remain:
```

Comment:
Fixed in latest commit.

Also, I can edit this again to reflect the 3.6 behaviour, but I'll have to merge this to master and cherry-pick it to 3.5+3.4 first.